src/layout/barPolar.js

```diff
@@ -75,15 +76,26 @@ function barLayoutPolar(seriesType, ecModel, api) {
         var barMinAngle = seriesModel.get('barMinAngle') || 0;
 
         lastStackCoords[stackId] = lastStackCoords[stackId] || [];
+        lastStackSum[stackId] = lastStackSum[stackId] || [];
 
         var valueDim = data.mapDimension(valueAxis.dim);
         var baseDim = data.mapDimension(baseAxis.dim);
         var stacked = isDimensionStacked(data, valueDim /*, baseDim*/);
         var clampLayout = baseAxis.dim !== 'radius'
             || !seriesModel.get('roundCap', true);
 
-        var valueAxisStart = valueAxis.getExtent()[0];
-
+        // valueAxisStart may not always be 0 consider of scale extent min and max
+        var radiusValueAxisStart;
+        if(valueAxis.scale.getExtent()[1] < 0) {
```

Comment:
Probably should consider `value` instead of extent. See https://github.com/apache/incubator-echarts/blob/c11c48343f6606fca65721a2eb65a7464791b592/src/layout/barGrid.js#L276